### ISSUE TITLE
Beams with setbacks totaling longer than the beam's length fail.

### ISSUE THREAD

**Issue by ikeough** [hypar-io/Elements#205]
The following snippet fails because the total length of the setbacks is equal to the length of the beam. This should be fixed by removing the setbacks when this occurs.
```cs
var line = new Line(Vector3.Origin, new Vector3(1,0,0));
var beam = new Beam(line, Polygon.Rectangle(0.5,0.5), BuiltInMaterials.Steel, 0.5, 0.5);
```

**Comment by ikeough** [comment]
```
There was an error executing the function.
The parameter t must be between 0.0 and 1.0.
   at Elements.Geometry.Line.PointAt(Double u) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Geometry/Line.cs:line 63
   at Elements.Geometry.Line.TransformAt(Double u, Double rotation) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Geometry/Line.cs:line 41
   at Elements.Geometry.Line.Frames(Double startSetback, Double endSetback, Double rotation) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Geometry/Line.cs:line 130
   at Elements.Geometry.Solids.Solid.SweepFaceAlongCurve(Polygon perimeter, IList`1 holes, ICurve curve, Double startSetback, Double endSetback, Double rotation) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Geometry/Solids/Solid.cs:line 119
   at Elements.Serialization.glTF.GltfExtensions.GetRenderDataForElement(GeometricElement e, Gltf gltf, Dictionary`2 materials, List`1 buffer, List`1 bufferViews, List`1 accessors) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Serialization/glTF/GltfExtensions.cs:line 630
   at Elements.Serialization.glTF.GltfExtensions.InitializeGlTF(Model model, List`1 buffer) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Serialization/glTF/GltfExtensions.cs:line 602
   at Elements.Serialization.glTF.GltfExtensions.SaveGlb(Model model, String path) in /Users/ikeough/Documents/Hypar/Hypar/Elements/src/Elements/Serialization/glTF/GltfExtensions.cs:line 538
   at Hypar.Commands.RunCommand.ExecuteFunction(String functionName, String functionDir, Boolean newBuild) in /Users/ikeough/Documents/Hypar/Hypar/CLI/src/Commands/RunCommand.cs:line 326
```

**Comment by ikeough** [comment]
Resolved in #205.
